crates/pallet-domains/src/lib.rs

```diff
@@ -211,6 +266,10 @@ mod pallet {
         DomainRuntimeUpgraded {
             runtime_id: RuntimeId,
         },
+        DomainInstantiated {
+            domain_id: DomainId,
+            runtime_id: RuntimeId,
```

Comment:
not sure if the `runtime_id` is useful here as this is part of the domain object, no ?

Reply:
Runtime event is used to send notifications about changes or conditions in the runtime to external entities like users, chain explorers, or dApps. I think `runtime_id` is important info about the domain instance that is worth delivery (just like `runtime_type` in `DomainRuntimeCreated` event).

Reply:
Not sure I agree here. Events should signal that domain was created with a specific DomainId. Rest of the information should be fetched from the state as required. If we add `runtime_id` then we should add `runtime_type` as well as just `runtime_id` is useless by itself.

Reply:
Okay, I don't have strong opinion here, the `runtime_id` is removed now PTAL.